op-bindings/Makefile

```diff
@@ -15,16 +17,19 @@ compile:
 		forge clean && \
 		pnpm build
 
-bindings: compile bindings-build
-
-bindings-build:
-	go run ./gen/main.go \
-		-forge-artifacts $(contracts-dir)/forge-artifacts \
-		-out ./bindings \
-		-contracts ./artifacts.json \
-		-source-maps MIPS,PreimageOracle \
-		-package $(pkg) \
-		-monorepo-base $(monorepo-base)
```

Comment:
For backwards compatibility, I think we should keep `bindings` and `bindings-build`, aliasing them to `bindgen-local` and `bindgen-generate-local` respectively.

eg
```
bindings: bindgen-local
bindings-build: bindgen-generate-local
```